Tests/Approval.Tests/ApprovedApi/FluentAssertions/net47.verified.txt

```diff
@@ -1044,6 +1048,11 @@ namespace FluentAssertions.Equivalency.Steps
         public StructuralEqualityEquivalencyStep() { }
         public FluentAssertions.Equivalency.EquivalencyResult Handle(FluentAssertions.Equivalency.Comparands comparands, FluentAssertions.Equivalency.IEquivalencyValidationContext context, FluentAssertions.Equivalency.IValidateChildNodeEquivalency valueChildNodes) { }
     }
+    public class TypeEquivalencyStep : FluentAssertions.Equivalency.IEquivalencyStep
+    {
+        public TypeEquivalencyStep() { }
+        public FluentAssertions.Equivalency.EquivalencyResult Handle(FluentAssertions.Equivalency.Comparands comparands, FluentAssertions.Equivalency.IEquivalencyValidationContext context, FluentAssertions.Equivalency.IValidateChildNodeEquivalency valueChildNodes) { }
+    }
```

Comment:
Should we keep `TypeEquivalencyStep` internal?
It will give us freedom to change/rename it later if we like.

Same question for `PredicateBasedTypingRule` below

Reply:
We usually don't so people can modify the step collection.